sdk/search/Azure.Search/src/SearchClientOptions.cs

```diff
@@ -106,7 +129,7 @@ private void AddLoggingHeaders()
         private void AddLoggingQueryParameters()
         {
             Diagnostics.LoggedQueryParameters.Add("api-version");
-            // TODO: Add the rest...
+            Diagnostics.LoggedQueryParameters.Add("$select");
```

Comment:
Might we want to add paging parameters? There's a lot of complexity in paging throughout the client, so it could be helpful. But I'm assuming we don't want to adding `$filter`, right (TMI)?

Reply:
We don't actually use any of those.  When sending via POST they all get serialized in the body.  $select on GetDocumentAsync is the only GET that that'll add things in the query params.

Reply:
`$select` contains field names, and we're currently debating whether they should be considered sensitive data, so maybe not a good idea to log them here?